### PR TITLE
Fix links in docs

### LINKED ISSUE #28287: Broken links in docs
### Description

image
Strange error in doc build. `en-us` also have this problem.
Why `administration/config-cheat-sheet.md` can not be detected here?

https://docs.gitea.com/development/api-usage
image

### Gitea Version

latest

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

gitea

### Database

None

### PR DESCRIPTION
**Pull request by yp05327**
Close #28287

## How to test it in local
convert Makefile L34 into:
```
cd .tmp/upstream-docs && git clean -f && git reset --hard && git fetch origin pull/28302/head:pr28302 && git switch pr28302
```

### CODE REVIEW

File: docs/content/installation/from-binary.zh-cn.md [hunk 1]
```diff
@@ -209,6 +209,6 @@ remote: ./hooks/pre-receive.d/gitea: line 2: [...]: No such file or directory
 
 如果您没有使用 Gitea 内置的 SSH 服务器，您还需要通过在管理选项中运行任务 `Update the '.ssh/authorized_keys' file with Gitea SSH keys.` 来重新编写授权密钥文件。
 
-> 更多经验总结，请参考英文版 [Troubleshooting](/en-us/install-from-binary/#troubleshooting)
```
Comment (yp05327) [inline docs/content/installation/from-binary.zh-cn.md:212]: I think this translation is not good. Are there any way to generate links for other language with same version?

**Comment by yp05327** [comment]
image
Awesome is not related. Only one issue mentioned above should be fixed.

**Comment by yp05327** [comment]
> title: "Fix docs" -> "Fix links in docs"?

Done.

**Review by lunny — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Comment by yp05327** [comment]
Backport?
image

**Comment by yp05327** [comment]
It seems that the bot won't work.
How to let bot work manually? 🤔

**Comment by yp05327** [comment]
emmm, where is the backport for v1.20?

**Comment by lunny** [comment]
> emmm, where is the backport for v1.20?

The teabot backport it to v1.20 failed.

**Pull request merged**